libs/wingsdk/test/testing/__snapshots__/simulator.test.ts.snap

```diff
@@ -149,7 +149,7 @@ exports[`run single test > test failure 1`] = `
 <sanitized>
 <sanitized>
     at Script.runInContext (node:vm:141:12)
-    at Object.runInContext (node:vm:297:6)
+    at Object.runInContext (node:vm:291:6)
```

Comment:
I started noticing these changes happening in my snapshots a few days ago. also a change in api from `keep-alive` to `close` 

not that its related to this PR but interesting to see its not just me (I have been uncomiting the files 😅 )

Reply:
Ah, yeah that's gross. I've just pushed a change that should fix this case hopefully